### ISSUE TITLE
`./configure` succeeds but `make` fails to build CoqIDE

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#15977]
#### Description of the problem
[log here](https://launchpadlibrarian.net/599308288/buildlog_ubuntu-impish-amd64.coq_8.master~git~202205010925+23207-0~daily375-d44d44b2d8~ubuntu21.10.1_BUILDING.txt.gz)
```
flock .dune.lock dune build --display=quiet --release _build/install/default/bin/coqide
    ocamlopt ide/coqide/.coqide_gui.objs/native/coqide.{cmx,o} (exit 2)
(cd _build/default && /usr/bin/ocamlopt.opt -w -40 -rectypes -g -O3 -unbox-closures -I ide/coqide/.coqide_gui.objs/byte -I ide/coqide/.coqide_gui.objs/native -I /usr/lib/ocaml/cairo2 -I /usr/lib/ocaml/lablgtk3 -I /usr/lib/ocaml/lablgtk3-sourceview3 -I /usr/lib/ocaml/threads -I boot/.boot.objs/byte -I boot/.boot.objs/native -I clib/.clib.objs/byte -I clib/.clib.objs/native -I config/.config.objs/byte -I config/.config.objs/native -I ide/coqide/.core.objs/byte -I ide/coqide/.core.objs/native -I ide/coqide/protocol/.protocol.objs/byte -I ide/coqide/protocol/.protocol.objs/native -I lib/.lib.objs/byte -I lib/.lib.objs/native -intf-suffix .ml -no-alias-deps -o ide/coqide/.coqide_gui.objs/native/coqide.cmx -c -impl ide/coqide/coqide.ml)
File "ide/coqide/coqide.ml", line 1364, characters 2-46:
1364 |   GtkData.StyleContext.add_provider_for_screen
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error: Unbound module GtkData.StyleContext
```

#### Coq Version

master, probably a result of #15964, cc @jfehrle

**Comment by jfehrle** [comment]
I think it's not related to #15964, which is only 4 lines and doesn't use any new calls from GTK. Take a look at it.

**Comment by JasonGross** [comment]
Ah, you're right, I was too quick to assume that because the most recently merged PR was about CoqIDE and the problem was not present the previous day, it was about this PR.  But the line of code was added 9 months ago...

**Comment by silene** [comment]
This is more likely a consequence of #15399, which requires lablgtk 3.1.2, while Ubuntu Impish only provides 3.1.1.

**Comment by silene** [comment]
This should be fixed by #15967, that is, `configure` will then fail.

**Comment by SkySkimmer** [comment]
#15967 merged
